lib/shopify-cli/shopifolk.rb

```diff
@@ -10,19 +10,35 @@ class Shopifolk
     SECTION = 'core'
     FEATURE_NAME = 'shopifolk'
 
-    ##
-    # will return if the user appears to be a Shopify employee, based on several heuristics
-    #
-    # #### Returns
-    #
-    # * `is_shopifolk` - returns true if the user is a Shopify Employee
-    #
-    # #### Example
-    #
-    #     ShopifyCli::Shopifolk.check
-    #
-    def self.check
-      ShopifyCli::Shopifolk.new.shopifolk?
+    class << self
+      attr_writer :acting_as_shopify_organization
+
+      ##
+      # will return if the user appears to be a Shopify employee, based on several heuristics
+      #
+      # #### Returns
+      #
+      # * `is_shopifolk` - returns true if the user is a Shopify Employee
+      #
+      # #### Example
+      #
+      #     ShopifyCli::Shopifolk.check
+      #
+      def check
+        ShopifyCli::Shopifolk.new.shopifolk?
+      end
+
+      def act_as_shopify_organization
+        @acting_as_shopify_organization = true
+      end
+
+      def acting_as_shopify_organization?
+        !!@acting_as_shopify_organization || (Project.has_current? && Project.current.config['shopify_organization'])
+      end
+
+      def reset
```

Comment:
This is so that we can clear the state in the tests.

Reply:
Just curious, could we not manually reset the state from the test instead of wrapping it in a method?

Reply:
Yes, we could with `ShopifyCli::Shopifolk.instance_variable_set(:@acting_as_shopify_organization, nil)`

But as `@acting_as_shopify_organization` isn't part of the public internal, this means we're reaching into its internal implementation.

Generally I would avoid adding methods that only exist for testing, but in situation I think it's acceptable.